tests/test_http_monitor.py

```diff
@@ -0,0 +1,105 @@
+"""Test suite for monitoring_scripts.http_monitor"""
+
+import pytest
+import requests
+
+from monitoring_scripts import http_monitor
```

Comment:
I'd recommend importing that module with `as unit`, so that you'll be able at a glance to know if you're interacting with the unit being tested.

It's a personal preference though.